ci-operator/step-registry/openshift-qe/rosa-upgrade-postcheck/openshift-qe-rosa-upgrade-postcheck-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: openshift-qe-rosa-upgrade-postcheck
```

Comment:
This can be removed too after we have `openshift-qe-rosa-loaded-upgrade-and-postcheck` chain.

Reply:
Update in new code, thanks!

Reply:
@liqcui I still see this in the new code.